plugins/connectrpc/kotlin/v0.5.1/buf.plugin.yaml

```diff
@@ -0,0 +1,29 @@
+version: v1
+name: buf.build/connectrpc/kotlin
+plugin_version: v0.5.1
+source_url: https://github.com/connectrpc/connect-kotlin
```

Comment:
We don't usually add patch releases for previous versions but protobuf-java 4.x had major breaking changes, and some users need to stay on the previous release until other dependencies can be upgraded.